src/build.gradle

```diff
@@ -3,22 +3,21 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath('gradle.plugin.com.hierynomus.gradle.plugins:license-gradle-plugin:0.16.1') {
+        classpath (libs.licenseGradlePlugin) {
             exclude group: 'org.springframework', module: 'spring-core'
         }
     }
 }
 
 plugins {
-    id 'org.sonarqube' version '5.0.0.4638'
-    id 'org.owasp.dependencycheck' version '9.0.10'
+    alias(libs.plugins.sonarqube)
+    alias(libs.plugins.dependencycheck)
     id 'jacoco-report-aggregation'
     id 'java'
     id 'idea'
-    id "io.spring.dependency-management" version "${springDependenciesVersion}" apply false
-    id 'org.springframework.boot' version "${springBootVersion}" apply false
-    id 'com.google.protobuf' version "$protobufGradleVersion" apply false
-    id 'org.siouan.frontend-jdk17' version "$frontendPluginVersion" apply false
+    alias(libs.plugins.springBoot) apply false
+    alias(libs.plugins.protobuf) apply false
+    alias(libs.plugins.frontendJDK17) apply false
```

Comment:
`apply false` pattern was used to version control dependencies. With version catalog it should not be necessary .